docs/pages/guides/liveblocks-pricing-model.mdx

```diff
@@ -0,0 +1,129 @@
+---
+meta:
+  title: "Liveblocks 1.0 pricing model"
+  parentTitle: "Guides"
+  description: "Explanation of Liveblocks 1.0 pricing model and implementation"
+---
+
+With the release of Liveblocks 1.0, we included new pricing plans that are based
+on monthly active users (MAU). This guide will help you understand how to use
+the new pricing plans by providing guidance on the migration to Liveblocks 1.0
+and how to include a `userId` into the `authorize` option if you use it.
+
+## What is a Monthly Active User (MAU) vs. a connection? [#mau-vs-connection]
+
+In versions before 1.0 we used the number of connections to determine pricing
+segments. We counted each time a user connected to a Liveblocks room from a
+physical device as a connection. While this approximated the number of users, it
```

Comment:
It never approximated the number of users. Dialogue has 500 MAU, and 4M connections. The whole reason why we did MAU is to map the pricing to the number of users